js/main.js

```diff
@@ -1 +1,76 @@
 /* Javascript goes here! */
+console.log('is your script running? better go catch it!');
+
+$(document).ready(function(){
+  console.log('ready');
+
+//I DONT KNOW WHAT TO DO TO MAKE MY WEBSITE DELETE OLD STUFF ON ENTRY!!!
```

Comment:
You can try to just clear out the text in those inputs by setting it to an empty string `$(input).text('')`.

Reply:
that worked; thank you. i tried my  ```.remove()``` functions again, and apparently they work now :smile: